AB_Hands_On/.ipynb_checkpoints/AB_Hands_On-checkpoint.ipynb

```diff
@@ -0,0 +1,6615 @@
+{
```

Comment:
<p>Didn't we want to add a section where we add the second B conversions and compare them?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='76'/>

Reply:
Youre not seeing it because youre reviewing the checkpoint file, not the actual notebook. The new section is included